packages/asc-core/src/components/Radio/RadioInput.ts

```diff
@@ -0,0 +1,30 @@
+import styled from '../../styled-components'
+
+type Props = {
+  id: string
+  name: string
+  label: string
+  value?: string
+  defaultChecked?: boolean
+  disabled?: boolean
+}
+
+const Radio = styled.input.attrs<Props>(props => ({
+  type: 'radio',
```

Comment:
Why do you need to define the styled component's attributes when the component already has props set?